scripts/qa_crowbarsetup.sh

```diff
@@ -2806,6 +2873,13 @@ function deploy_single_proposal
                 return
             fi
             ;;
+        ironic)
+            [[ $want_ironic = 1 ]] || return
+            if ! iscloudver 7plus; then
+                echo "Ironic is SOC 7+ only. Skipping"
```

Comment:
Add a 'Warning: ' prefix here so that it gets picked up by the jenkins log parser